doc/content/reference/email-templates/overriding.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Overriding Templates"
+description: ""
+weight: 6
+---
+
+> Note: We recommend getting familliar with the [html/template](https://golang.org/pkg/html/template/) template format first.
+
+## Template components
+
+An email template override has 3 components:
+
+1. The subject file, which contains the subject of the email and is named `<identifier>.subject.txt`. 
+2. The text contents file, which contains the contents of the email in text format and is named `<identifier>.txt`.
+3. The HTML contents file, which contains the contents of the email in HTML format and is named `<identifier>.html`.
+
+
+## Creating the overrides
+
+In order to override a template, one must provide all 3 files as part of the email templates repository and then provide them to a [fetcher]({{< ref "/concepts/fetchers.md" >}}).
+
+Let's consider that we want to override the email that a user receives once he registers, the validation email (`validate`). We need to create the following files:
+
+File name | Contents
+----------|---------
+validate.subject.txt | Please confirm your email address for {{.Network.Name}}
+validate.txt | Please confirm your email address for {{.Network.Name}}. <br> Your email address will be used as contact for {{.Entity.Type}} "{{.Entity.ID}}". <br><br> Reference: {{.ID}} <br> Confirmation Token: {{.Token}}
+validate.html | Please confirm your email address for {{.Network.Name}}. &lt;br&gt;&lt;br&gt; <br> Your email address will be used as contact for {{.Entity.Type}} "{{.Entity.ID}}". &lt;br&gt;&lt;br&gt; <br> Reference: {{.ID}} &lt;br&gt; <br> Confirmation Token: {{.Token}}
+
+## Providing the overrides to the stack
+
+Once you have written your overrides, you can provide them to the stack either through an remote URL, or through the local file system.
+
+```yaml
```

Comment:
I don't think we should do this here, otherwise we also need to document the corresponding flags/env.

Reply:
Any other thoughts about how this should look ? I mentioned the `yaml` configuration since the getting started guide already mentions it for the CLI.

Reply:
I do think we need some reference to configuration. When reading the docs I was curious to _how_ to configure this really.

Reply:
@johanstokking I agree, but doesn't the configuration reference belong in a **Configuration** section in the **Reference** section, instead of the **Email Templates**?

Reply:
What do we put there, information about configuration in general?

I think, at minimum, we should mention the concerning yaml and env keys in a reference. We should not redirect to generic configuration where people have to figure out the potentially relevant keys out of all the possible keys.